Docs/releases/0.16.2.md

```diff
@@ -0,0 +1,9 @@
+## Changes in `0.16.2`
+
+***This is a bug fix release!***
+
+Check out the [0.16.2 milestone](https://github.com/HubSpot/Singularity/issues?q=milestone%3A%0.16.2+is%3Aclosed) to see pull requests in detail.
```

Comment:
I think the link should be `https://github.com/HubSpot/Singularity/issues?q=milestone%3A0.16.2+is%3Aclosed` 
Seems to be an extra % before the 0.16.2